src/components/ResponsiveGrid.jsx

```diff
@@ -0,0 +1,25 @@
+// ============================================================================
+// IMPORTS
+// ============================================================================
+
+import PropTypes from "prop-types";
+
+function ResponsiveGrid({ minColumnWidth, rowGap, children }) {
+  const style = {
+    display: "grid",
+    gridTemplateColumns: `repeat(auto-fit, ${minColumnWidth})`,
+    rowGap,
+    placeItems: "center start",
+    minHeight: "fit-content"
+  };
+
+  return <div style={style}>{children}</div>;
+}
+
+ResponsiveGrid.propTypes = {
+  minColumnWidth: PropTypes.string.isRequired,
+  rowGap: PropTypes.string.isRequired,
+  children: PropTypes.node
+};
```

Comment:
Consider setting default values for minColumnWidth and rowGap in case they are not provided by the parent component.

Reply:
`minColumnWidth` and `rowGap` are required properties.

*This component needs to be documented.*